docs_source_files/content/webdriver/browser_manipulation.de.md

```diff
@@ -1143,7 +1143,15 @@ driver.save_screenshot('./image.png')
 driver.quit()
 
 {{< / code-panel >}}
-  {{< code-panel language="csharp" >}} // Code sample not available please raise a PR {{< / code-panel >}}
+  {{< code-panel language="csharp" >}} 
+    using OpenQA.Selenium;
+    using OpenQA.Selenium.Chrome;
+    using OpenQA.Selenium.Support.UI;
+
+    var driver = new ChromeDriver();
+    var screenshot = (driver as ITakesScreenshot).GetScreenshot();
```

Comment:
I think need to navigate to a url like driver.get("http://www.example.com"); before capturing screenshot

Am not sure how driver builder works in C# if below code
```
 var driver = new ChromeDriver();
```
Opens a blank page, then the above code is valid. Else we need to navigate to URL and take screenshot as other code samples do

Reply:
Oh, that's right. This is the code for taking screenshot of an already opened browser which I attached to Webdriver via changing the debugAddress in options. For a new blank page navigating to some url and then screenshotting it, I'll make that change in the code.